examples/TestDriver/android/app/build.gradle

```diff
@@ -146,6 +146,7 @@ android {
 }
 
 dependencies {
+    implementation 'com.heapanalytics.android:heap-android-client:0.8.3'
```

Comment:
You don't want to add this here. You want to modify `react-native-heap/android/build.gradle` to have the following line:
`api 'com.heapanalytics.android:heap-android-client:0.8.3'`

This would replace the `implementation... line`.

See [here](https://developer.android.com/studio/build/dependencies#dependency_configurations). Try that out locally and test it to see if it does what you want.